Sample_v3/Extensions/Channel+MembersInfo.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChatClient
+
+extension Channel {
+    var membersInfo: String {
+        "\(members.count) members, \(members.filter(\.isOnline).count) online"
+    }
+    
+    var typingMembersInfo: String? {
+        guard !currentlyTypingMembers.isEmpty else { return nil }
+        let names = currentlyTypingMembers.map { $0.name ?? $0.id }.sorted()
+        return names.joined(separator: ",") + " \(names.count == 1 ? "is" : "are") typing..."
+    }
+}
```

Comment:
Yes, I remember we discussed that the sample app has to be straight forward. 
But if we didn't extract this part it would be duplicated in 6 places 😄

Reply:
I guess my main problem is with having it as an extension of `Channel`. It's not clear what's the official API and what is a sample-specific extension.

Reply:
We might consider moving this extension to the SDK but it contains non-localized strings. Not sure someone is gonna use these methods in production